test/namespace_admin_test.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+from typing import Callable, Optional
+
+from test_base import TestBase
+
+from warnet.k8s import get_kubeconfig_value
+from warnet.process import run_command
+
+
+class NamespaceAdminTest(TestBase):
+    def __init__(self):
+        super().__init__()
+        self.namespace_dir = (
```

Comment:
the test in https://github.com/bitcoin-dev-project/warnet/pull/622 would be a good model for this one as well. after running the test locally I end up with a `kubeconfigs/` directory in the repo root. The test should operate in /tmp

Reply:
Thank! Fixed in https://github.com/bitcoin-dev-project/warnet/pull/616/commits/64d1b40bd4f9bd345cb481dcdaa875e08c1b00c0